conman/routes/handlers.py

```diff
@@ -32,6 +38,22 @@ class URLConfHandler(BaseHandler):
     Views referenced in the `urlconf` will receive `route`, as well as the args
     and kwargs they would expect given their urlpattern.
     """
+    @classmethod
+    def check(cls, route):
+        """Ensure route has a sensible urlconf attribute."""
+        route_name = route.__name__
+        if not hasattr(route, 'urlconf'):
+            return [checks.Error(
+                '{} must have a `urlconf` attribute.'.format(route_name),
```

Comment:
Good point about the variable. My copy-pasting is showing through `:P`

`__qualname__` can't hurt either. I'll pop that in there.